app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -43,12 +43,17 @@ class FractionInteractionViewModel private constructor(
         override fun onPropertyChanged(sender: Observable, propertyId: Int) {
           errorOrAvailabilityCheckReceiver.onPendingAnswerErrorOrAvailabilityCheck(
             pendingAnswerError,
-            answerText.isNotEmpty()
+            true // Allow submit on empty answer
           )
         }
       }
     errorMessage.addOnPropertyChangedCallback(callback)
     isAnswerAvailable.addOnPropertyChangedCallback(callback)
+    // Apply defaults:
```

Comment:
Could you expand in this comment on why the default call below is needed? It's not obvious to me as a reader, so I suspect it will also be unclear for future folks who come across this code in isolation (vs. in the context of the broader PR).

Reply:
Updated the doc